src/controller/file/serializing/filter_serialization.py

```diff
@@ -45,7 +45,7 @@ def _create_filter_element_for_fish(filter_: Filter, parent: ElementTree.Element
             _create_filter_configuration_element(filter_configuration=filter_configuration, parent=filter_element)
 
 
-def create_channel_mappings_for_filter_set_for_fish(om: SceneOptimizerModule, scene_element: ElementTree.Element):
+def create_channel_mappings_for_filter_set_for_fish(for_fish, om: SceneOptimizerModule, scene_element: ElementTree.Element):
```

Comment:
We don't need this argument here as this function will only be called if we're assembling the filters for fish.

Reply:
We'll leave it the way it is, as discussed.